From a4ff4b5e952a22906ac0ed01e73f84f42926c325 Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Thu, 24 Nov 2022 14:29:06 -0700
Subject: [PATCH] Fix FindFlatBuffers cmake file

Capitalization needs to match for system cmake override to work:
https://github.com/google/flatbuffers/blob/v22.11.23/CMake/FindFlatBuffers.cmake

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[Upstream status:
https://github.com/tensorflow/tensorflow/pull/58677]
---
 tensorflow/lite/CMakeLists.txt                                  | 2 +-
 .../modules/{FindFlatbuffers.cmake => FindFlatBuffers.cmake}    | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename tensorflow/lite/tools/cmake/modules/{FindFlatbuffers.cmake => FindFlatBuffers.cmake} (100%)

diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
index f9c30d6a046..c2ee9edfb61 100644
--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -143,7 +143,7 @@ find_package(absl REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(farmhash REQUIRED)
 find_package(fft2d REQUIRED)
-find_package(Flatbuffers REQUIRED)
+find_package(FlatBuffers REQUIRED)
 find_package(gemmlowp REQUIRED)
 find_package(NEON_2_SSE REQUIRED)
 find_package(cpuinfo REQUIRED)  #CPUINFO is used by XNNPACK and RUY library
diff --git a/tensorflow/lite/tools/cmake/modules/FindFlatbuffers.cmake b/tensorflow/lite/tools/cmake/modules/FindFlatBuffers.cmake
similarity index 100%
rename from tensorflow/lite/tools/cmake/modules/FindFlatbuffers.cmake
rename to tensorflow/lite/tools/cmake/modules/FindFlatBuffers.cmake
-- 
2.34.1

